content/docs/details/troubleshooting-the-catalog/index.md

```diff
@@ -6,32 +6,32 @@ description: How to resolve issues getting data into the catalog
 
 ## About Locations
 
-The Backstage catalog uses location entities to point to external sources which define entities. Roadie will 
+The Backstage catalog uses location entities to point to external sources which define entities. Roadie will
 update your entities in Backstage whenever one of the repositories defined in a location is updated.
 
-While locations are themselves entities they are generally considered internal to the workings of Backstage and not part 
+While locations are themselves entities they are generally considered internal to the workings of Backstage and not part
 of your catalog (though you may need to create locations directly in some use cases).
 
 ## Using the Locations log
 
 The locations log provides information on the scheduling of the catalog refresh cycle and surfaces any errors.
 
-![locations log home](./locations-log-unsorted.png)
+![locations log home](./locations-log.png)
 
 ### Searching for an entity
 
-You can search for an entity using the filter input (Top right of the table). The filter operates on the `entity/location url`
-column. The values in this column are [entity refs](https://backstage.io/docs/features/software-catalog/references#string-references) (e.g. kind:namespace/entity-name) 
-or for a location entity the target URL. In practice you'll want to search on either your entity name or perhaps
-the URL of a catalog info YAML file. To view details for a particular entity click the entity link.
+You can search for an entity using the filter input (Top right of the table). The filter operates on the `Entity ref`
+column and you can search for the `spec.target` field of Location entities. The values in this column are [entity refs](https://backstage.io/docs/features/software-catalog/references#string-references) (e.g. kind:namespace/entity-name)
```

Comment:
I'm thinking new users might not know what the spec.target field is? perhaps an example? And perhaps we should lead with that rather than entity ref? (as it's generally fairly opaque)